imgproxy/templates/deployment.yaml

```
@@ -15,7 +15,7 @@ spec:
       app: {{ template "imgproxy.fullname" . }}
   template:
     metadata:
-      labels:
+      labels: {{ toYaml .Values.pod.lables | nindent 8 }}
```

Hi @123BLiN
Could you fix this line and check the result `helm lint imgproxy && helm template imgrpoxy --debug --generate-name --dry-run`?

```suggestion
      labels: {{ toYaml .Values.pod.labels | nindent 8 }}
```